flyway-commandline/src/main/java/org/flywaydb/commandline/logging/version/VersionCommandExtension.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.flywaydb.commandline.command.version;
+package org.flywaydb.commandline.logging.version;
```

Comment:
Hope I'm within the bounds of reasonableness. Since you've changed the path here to be `logging` instead of `version`, you also need to update the line in `src/main/resources/META-INF/services/org.flywaydb.core.extensibility.Plugin` to `org.flywaydb.commandline.logging.version.VersionCommandExtension` as well. (or at least I had to in order to build this)

I think I'm missing some context here, but what's the motivation behind moving these files into `logging`?